slices/base-passwd.yaml

```diff
@@ -0,0 +1,14 @@
+package: base-passwd
+
+slices:
+  data:
+    contents:
+      /usr/share/base-passwd/group.master:  {until: mutate}
+      /usr/share/base-passwd/passwd.master: {until: mutate}
+      /etc/group:  {text: FIXME, mutable: true}
+      /etc/passwd: {text: FIXME, mutable: true}
+    mutate: |
+      pw = content.read("/usr/share/base-passwd/group.master")
```

Comment:
this is probably the world most nitpciking commend :p , but the var names are swapped. `pw` should be `gr` and vice versa

Reply:
Good catch :-D